.github/workflows/test_comprehensive.yml

```diff
@@ -62,6 +62,15 @@ jobs:
           - python: 3.8
             platform: ubuntu-20.04
             toxenv: headless-py38-linux
+          - python: 3.11
+            platform: ubuntu-latest
+            backend: pyqt6
+          - python: 3.11
+            platform: ubuntu-latest
+            backend: pyside6
+        exclude:
+          - python: "3.11"
+            backend: pyside2
```

Comment:
@Czaki why is it `3.11` in some sections and `"3.11"` in others? If possible, it would be good to be consistent.

Reply:
It should be `"3.11"` everywhere for consistency with 3.10

Reply:
would you like me to push that change to the PR?

Reply:
Ok I'm a-pushing! 😅